IntegrationTests/CarthageIntegration/CarthageIntegration.xcodeproj/project.pbxproj

```diff
@@ -476,7 +476,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
🙏  thank you!

Reply:
![nodding](https://user-images.githubusercontent.com/1304821/139950193-d3478589-de12-4684-8cfa-06ee337591da.gif)